Makefile

```diff
@@ -285,11 +285,9 @@ up-detach: build-services ## Bring up local Grapl and detach to return control t
 		up --detach --force-recreate
 
 .PHONY: down
-down: ## docker-compose down - both stops and removes the containers
-	$(WITH_LOCAL_GRAPL_ENV)
-	docker-compose down --timeout=0
-	docker-compose --project-name $(COMPOSE_PROJECT_INTEGRATION_TESTS) down --timeout=0
-	docker-compose --project-name $(COMPOSE_PROJECT_E2E_TESTS) down --timeout=0
+down: ## Remove all Grapl containers
+	# Remove all Grapl containers - continue on error (no containers found)
+	docker rm --volumes --force $$(docker ps --filter "name=grapl*" --all --quiet) 2>/dev/null || true
```

Comment:
Strictly speaking, you don't need the `*` in the filter; it treats the value as a prefix already.

Reply:
A side effect of going away from `docker-compose down` is that this will leave the `grapl-network` docker network behind.

Reply:
Ah yes, the network being left behind is a good point.

Reply:
A possibly-equivalent solution would be to filter on the network the container is attached to:

```
docker rm --volumes --force $(docker ps --filter "network=grapl-network" --all --quiet)
docker network rm grapl-network
```

Reply:
I'm fine with using `docker-compose down` if you'd prefer. It'd mean things like networks and such would be handled automatically.

Reply:
Yeah, I've got no problem with the `docker-compose down`-based approach. I'm cool with sticking to that.